packages/components/src/__future__/Tag/_docs/Tag.mdx

```diff
@@ -0,0 +1,33 @@
+import { Canvas, Controls, Meta } from "@storybook/blocks"
+import { ResourceLinks, KaioNotification, Installation } from "~storybook/components"
+import * as TagStories from "./Tag.stories"
+
+<Meta of={TagStories} />
+
+# Avatar
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/__future__/Tag"
+  figma="https://www.figma.com/file/ZRfnoNUXbGZv4eVWLbF4Az/%F0%9F%96%BC%EF%B8%8F-Component-Gallery?type=design&node-id=9-37843&mode=design&t=v8aGjXZU6apybb6y-0"
+  designGuidelines="https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3293283155/Tag"
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
+<Installation
+  installCommand="yarn add @kaizen/components"
+  importStatement="import { Tag } from `@kaizen/components/future`"
+/>
+
+## Overview
+
+Tags helps users quickly recognize important information about items that organize and categorize them.
+
```

Comment:
We'll need to add a note here that this `Tag` is not yet feature complete and consumers will need to use the Legacy `Tag` if they still need the legacy features.

The goal of this `Tag` was to first provide a flexible option compared to the Legacy one, and then we will slowly add more functionality until we can completely replace the Legacy version.